README.md

```diff
@@ -83,6 +84,8 @@ More information about these configurations can be found in the [Deploy Sematic
 | `deployment.autoscaling.target_memoryu_utilization_pct`   | Target memory utilization for autoscaled Sematic server pods  | Not set                                 |
 | `deployment.security_context`                             | Pod security context for Sematic server                       | `{}`                                    |
 | `deployment.security_context.fs_group`                    | Pod security filesystem group for Sematic server              | Not set                                 |
+| `deployment.socket_io.dedicated`                          | Starts a separate dedicated Sematic server for Socket I/O     | `false`                                 |
+| `deployment.socket_io.address`                            | Cluster-private URL to reach dedicated Socket I/O server      | Not set                                 |
```

Comment:
This was removed from the values definitions. Please remove this doc entry as well.